EIPS/eip-history-expiry-meta.md

```diff
@@ -0,0 +1,115 @@
+
+### Devnet Activation
+
+Execution clients may test dropping of history on devnets for all history prior to block `TODO-WHAT-BLOCK-NUMBER?`.
+
+
+## Rationale
+
+### Why wait for Pecra
+
+Consensus Layer clients have a dependency on pre-merge deposit logs. [EIP-6110](https://eips.ethereum.org/EIPS/eip-6110) will remove this dependency when the Pectra fork is activated.
+
+### Why drop Sepolia history
+
+The Sepolia history drop is intended as a testing ground for the mainnet activation.
+
+### Why drop Devnet history
+
+The Devnet history drop is intended to test prior to Sepolia to avoid any breakage on the Sepolia network.
+
+### Won't this break JSON-RPC
+
+History Expiry doesn't require clients to remove this data. It only allows them to. Clients that wish to preserve this history in their client for JSON-RPC use cases are free to do so.
+
+
+### Where will Pre-Merge history be stored
+
+Pre-merge data is available in the [e2store archival format](https://github.com/eth-clients/e2store-format-specs). A public list of these archives can be found in the [eth-clients historical data endpoints](https://eth-clients.github.io/history-endpoints/)
+
+The Portal network also implements a decentralized peer-to-peer solution for storage and retrieval of all of Ethereum's pre-merge block data.
+
+
+## Backwards Compatibility
+
+### DevP2P `eth` protocol
+
+Clients of the DevP2P `eth` protocol will need to upgrade to the new `eth/69` version specified in [EIP-7642](https://eips.ethereum.org/EIPS/eip-7642)
+
+### Pre-Merge Deposit Logs
+
+Consensus Layer clients have had a historical dependency on the deposit logs from pre-merge blocks. Dropping history would make these logs inaccessible to the Consensus Layer client. This issue is mitigated by [EIP-6110](https://eips.ethereum.org/EIPS/eip-6110)
+
+### Serving Pre-Merge JSON-RPC
+
+Execution clients that choose to drop history will no longer be capable of serving JSON-RPC requests for pre-merge requests for the following endpoints without sourcing the data from an alternate data source.
+
+- `eth_getBlockTransactionCountByHash`
+- `eth_getBlockTransactionCountByNumber`
+- `eth_getUncleCountByBlockHash`
+- `eth_getUncleCountByBlockNumber`
+- `eth_getBlockByHash`
+- `eth_getBlockByNumber`
+- `eth_getTransactionByHash`
+- `eth_getTransactionByBlockHashAndIndex`
+- `eth_getTransactionByBlockNumberAndIndex`
+- `eth_getTransactionReceipt`
+- `eth_getUncleByBlockHashAndIndex`
+- `eth_getUncleByBlockNumberAndIndex`
+
+
+## Security Considerations
+
+### Full History Sync
+
+Execution layer clients will no longer be able to implement a full historical sync of history from the DevP2P network.  Clients that wish to retain this functionality will need to source the pre-merge blocks from an alternate source.  Clients SHOULD ensure that they continue to correctly validate block data sourced from alternate locations.
```

Comment:
that should still be possible with 7801 which uses DevP2P

Reply:
I'm gonna add a section above about 7801 to the list of alternate sources for this data, and be more specific here that this is about the `eth` protocol.